modules/library/render/src/main/java/org/geotools/renderer/lite/StreamingRenderer.java

```diff
@@ -1996,114 +1892,68 @@ public int getMaxBackBufferMemory(int width, int height) {
      * @throws IOException
      * @throws IllegalFilterException
      */
-    private void processStylers(final Graphics2D graphics,
-            final Layer layer, AffineTransform at,
-            CoordinateReferenceSystem destinationCrs, Envelope mapArea,
-            Rectangle screenSize, String layerId) throws Exception {
-        /*
-         * DJB: changed this a wee bit so that it now does the layer query AFTER
-         * it has evaluated the rules for scale inclusion. This makes it so that
-         * geometry columns (and other columns) will not be queried unless they
-         * are actually going to be required. see geos-469
-         */
+    private void processStylers(final Graphics2D graphics, final Layer layer, String layerId)
+            throws Exception {
```

Comment:
javadocs no longer match the method signature. Yeah I know it is a private method but it is an important one.

```
      * @param graphics Target graphics for rendering
      * @param layer Layer styled
      * @param layerId Handle used to identify the layer
```